<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Project set up done]]>
    </line>
    <line>
      <![CDATA[Application up and running as expected]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="125390" started-at="2021-04-23T19:00:13Z" finished-at="2021-04-23T19:02:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="125390" started-at="2021-04-23T19:00:13Z" finished-at="2021-04-23T19:02:18Z">
      <class name="HFIC.Testcases.com.PEGA_Application_process">
        <test-method status="PASS" signature="Start_up()[pri:0, instance:HFIC.Testcases.com.PEGA_Application_process@2133c8f8]" name="Start_up" is-config="true" duration-ms="1317" started-at="2021-04-23T15:00:11Z" finished-at="2021-04-23T15:00:13Z">
          <reporter-output>
            <line>
              <![CDATA[Project set up done]]>
            </line>
          </reporter-output>
        </test-method> <!-- Start_up -->
        <test-method status="PASS" signature="Application_Start()[pri:0, instance:HFIC.Testcases.com.PEGA_Application_process@2133c8f8]" name="Application_Start" is-config="true" duration-ms="2856" started-at="2021-04-23T15:00:13Z" finished-at="2021-04-23T15:00:15Z">
          <reporter-output>
            <line>
              <![CDATA[Application up and running as expected]]>
            </line>
          </reporter-output>
        </test-method> <!-- Application_Start -->
        <test-method status="PASS" signature="PEGA_App_process()[pri:0, instance:HFIC.Testcases.com.PEGA_Application_process@2133c8f8]" name="PEGA_App_process" duration-ms="120051" started-at="2021-04-23T15:00:15Z" finished-at="2021-04-23T15:02:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PEGA_App_process -->
        <test-method status="PASS" signature="Application_Exit()[pri:0, instance:HFIC.Testcases.com.PEGA_Application_process@2133c8f8]" name="Application_Exit" is-config="true" duration-ms="2461" started-at="2021-04-23T15:02:15Z" finished-at="2021-04-23T15:02:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Application_Exit -->
        <test-method status="FAIL" signature="Tear_Down(org.testng.ITestResult)[pri:0, instance:HFIC.Testcases.com.PEGA_Application_process@2133c8f8]" name="Tear_Down" is-config="true" duration-ms="0" started-at="2021-04-23T15:02:18Z" finished-at="2021-04-23T15:02:18Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[wrong number of arguments]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.TestRunner.afterRun(TestRunner.java:1012)
	at org.testng.TestRunner.run(TestRunner.java:636)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tear_Down -->
      </class> <!-- HFIC.Testcases.com.PEGA_Application_process -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
